src/hoc/selectable-enhance.js

```diff
@@ -76,12 +76,20 @@ export const SelectableContainerEnhance = (Component) => {
           key: this._keyIndex,
           style: mergedChildrenStyles,
           nestedItems: child.props.nestedItems.map((child) => this._extendChild(child, styles, selectedItemStyle)),
+          initiallyOpen: child.props.nestedItems.reduce(this._hasSelectedDescendant, false),
```

Comment:
> it will be universally beneficial

That makes sense. It's similar to when we introduce breaking changes for following more closely the specification without following semver.
@alitaheri @newoga Are you ok with keeping it like this?